src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
The `font-weight: 400;` is set globally for all elements. If you intend to use different font weights for specific elements, ensure to override this property in those specific styles. Otherwise, all text will have the same weight, which might not be desirable for headings or emphasized text.